fv3gfs/wrapper/physics_properties.json

```diff
@@ -455,5 +462,26 @@
         "units": "unknown",
         "container": "Sfcprop",
         "dims": ["z_soil", "y", "x"]
+    },
+    {
+        "name": "total_sky_downward_longwave_flux_at_surface_override",
+        "fortran_name": "adjsfcdlw_override",
+        "units": "W/m^2",
+        "container": "Statein",
+        "dims": ["y", "x"]
+    },
+    {
+        "name": "total_sky_downward_shortwave_flux_at_surface_override",
+        "fortran_name": "adjsfcdsw_override",
```

Comment:
I believe the name for `adjsfcdsw` is something like `time_adjusted_total_sky_downward_shortwave_flux_at_surface`. Please confirm the "total sky" part. Similarly for the other `adj` names.

Reply:
Yes, probably good to put "time_adjusted" in there.  Yes, these are all "total_sky."